packages/core/src/browser/dialogs.ts

```diff
@@ -194,7 +194,9 @@ export abstract class AbstractDialog<T> extends BaseWidget {
         this.activeElement = undefined;
         super.close();
     }
-
+    error(error: string): void {
```

Comment:
please remove, there a contract that the validation callback should provide an error and it is called at the appropriate moment